packages/fast-components-react-base/src/image/image.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import Foundation, { HandledProps } from "../foundation";
+import { IImageHandledProps, IImageMangedClasses, IImageUnhandledProps } from "./image.props";
+import { IImageClassNameContract, IManagedClasses } from "@microsoft/fast-components-class-name-contracts";
+
+/**
+ * TODO #239 #240: Hook viewports values into Styles
+ */
+export interface IBreakpoints {
+    vp1: number;
+    vp2: number;
+    vp3: number;
+    vp4: number;
+    vp5: number;
+    vp6: number;
+}
+
+const breakpoints: IBreakpoints = {
+    vp1: 0,
+    vp2: 540,
+    vp3: 768,
+    vp4: 1084,
+    vp5: 1400,
+    vp6: 1779
+};
+
+/* tslint:disable-next-line */
+class Image extends Foundation<IImageHandledProps & IManagedClasses<IImageClassNameContract>,  React.HTMLAttributes<HTMLImageElement | HTMLPictureElement>, {}> {
+    protected handledProps: HandledProps<IImageHandledProps & IManagedClasses<IImageClassNameContract>> = {
+        managedClasses: void 0,
+        alt: void(0),
+        itemScope: void(0),
+        round: void(0),
+        src: void(0),
+        vp1: void(0),
+        vp2: void(0),
+        vp3: void(0),
+        vp4: void(0),
+        vp5: void(0),
+        vp6: void(0),
+    };
+
+    /**
+     * Renders the component
+     */
+    public render(): React.ReactElement<HTMLImageElement | HTMLPictureElement> {
+        if (this.props.src) {
```

Comment:
I think we made this change already, but adding here for others. If there is no src, the picture element render function will be called. Then we check for vp1 to render the nested <img>. If that's not there - I don't think it will show up. 

Instead, we can add a null check here and remove the ternary from picture:

```
if (!this.props.src || !this.props.vp1) { return }
```